README.md

```diff
@@ -7,38 +7,31 @@ A __Rust__ binding for the [GSL library](http://www.gnu.org/software/gsl/) (the
 
 This binding requires the [GSL library](http://www.gnu.org/software/gsl/) library to be installed.
 
-To build it, please use :
+This crate works with Cargo and is on
+[crates.io](https://crates.io/crates/GSL).
```

Comment:
I'm just wondering: why so much backlines ? Here, you could have let "[crates.io]..." on the same line, no ?